tests/lib/external/snapd-testing-tools/.github/workflows/tests.yaml

```diff
@@ -0,0 +1,30 @@
+name: Jobs
+on:
+  push:
+    branches:
+      - '**'        # matches every branch
+
+jobs:
+  unit-tests:
+    runs-on: ubuntu-20.04
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v2
+
+    - name: Run shellCheck for tools
+      run: |
+          sudo apt-get remove --purge shellcheck
+          sudo snap install shellcheck
+
+          find tools -type f -exec sh -c "head -n 1 {} | egrep -a 'bin/bash|bin/sh' >/dev/null" \; -print -exec shellcheck {} \;
```

Comment:
To not fail on file names with spaces, that should work  better:

`find tools -type f -exec sh -c 'head -n 1 "${1}" | egrep -a "bin/bash|bin/sh" >/dev/null' '--' {} \; -print -exec shellcheck {} \;`